public/js/components/file-browser/file-browser.html

```diff
@@ -65,7 +65,7 @@
               </svg></a>
           </td>
         </tr>
-        <tr ng-repeat-end ng-if="binding.epxanded">
+        <tr ng-repeat-end ng-if="binding.expanded">
```

Comment:
here is another possibly functionality effecting change - but my guess since was incorrectly typed everywhere -- should be fine?